.env.test

```diff
@@ -1 +1,2 @@
-BASIC_AUTH_PASS=
\ No newline at end of file
+BASIC_AUTH_PASS=
+DATABASE_URL=horizon_test
```

Comment:
I remember while we were pairing we talked about the value of `DATABASE_URL` and its relationship to one's local databases but I might not have done a great job of explaining myself! 😛 

By default a Rails application will use the database configuration found in the `config/database.yml` file. Only when a different type of setup is required should another configuration method be used. Setting `DATABASE_URL` is how we configure things in k8s, but I would NOT expect to see this in a local setup.

The docs do a pretty decent job running through things:

https://guides.rubyonrails.org/configuring.html#configuring-a-database

> If you have both config/database.yml and ENV['DATABASE_URL'] set then Rails will merge the configuration together.

Looking back at the config/database.yml file on master, I would expect it to work great for local development, including running tests! The wrinkle here is that if one has DATABASE_URL set in their .env file, then the test one will pick that up!! This is super dangerous!

And so what you've done here is great! Including the `DATABASE_URL` in the test version of this file prevents this particular mistake from occurring, but I just want to emphasize that using this environment var at all is super risky. A better approach to using staging/production data is the pull data task over in the bin folder. Pulling data from a remote DB into a local one ensures that while messing around with things, nothing upstream is affected.

Anyway, thanks for coming to my TED talk!! ❤️ ❤️ ❤️

Reply:
I am guilty of connecting to staging locally, and want to guard against anyone who sets that locally -- going to add a comment warning not to use it.

Reply:
Perfect!